lib/src/empire_property.dart

```diff
@@ -207,6 +207,7 @@ class EmpireProperty<T> implements EmpireValue<T> {
   }
 
   @override
+  // ignore: non_nullable_equals_parameter
```

Comment:
There was a new check / warning added by the flutter team to the default analysis options. Ignoring as this is the intended behaviour we want / need in order to evaluate equality between empire property types.